include/caffe/solver.hpp

```diff
@@ -107,6 +107,7 @@ class Solver {
   virtual void RestoreSolverStateFromHDF5(const string& state_file) = 0;
   virtual void RestoreSolverStateFromBinaryProto(const string& state_file) = 0;
   void DisplayOutputBlobs(const int net_id);
+  void updateSmoothedLoss(Dtype loss, int start_iter, int average_loss);
```

Comment:
This should be `UpdateSmoothedLoss` to be consistent with other method names.

(I personally don't like the caffe capitalization style, but consistency is important)